strapi/strapi
# [ADMIN] Content Manager improvement while adding new entry

### What does it do?

In Content Manager, added new button in header named `Add another entry` when someone save the new Collection Type entry.

### Why is it needed?

When creating a new Content Type, users often need to add multiple entries. Currently, after adding an entry, users have to navigate back to the `Content Manager > Collection Type` page and click on `Create new entry` to add another one.

This PR enhances the user experience by allowing users to directly navigate to the "Create new entry" page after adding an entry. This eliminates the need to go back to the Content Type List page, saving valuable time, especially when adding a large number of contents to the Content Type.

With this improvement, users can seamlessly continue adding entries without unnecessary navigation, streamlining their workflow and enhancing productivity.

### How to test it?

- Login to admin
- Navigate to Content Manager
- Add any entry under any Content Type
- Once added you can see `Add another entry` button on top right 
- By clicking `Add another entry` button you can easily add another entry without navigating back to Content Type lists.

https://github.com/strapi/strapi/assets/40918526/162450f9-782d-4a4c-a225-dbceff2421cd



### Related issue(s)/PR(s)

This PR is not related to any open issue.

**PS:** I have included translations for all available languages using Google Translator. I have tried my best to ensure accurate translations, but please let me know if there are any specific changes needed for a particular language. I'm more than happy to make updates and ensure the translations are correct. Your feedback is greatly appreciated!
